sdk/golang/ccnp/quote/proto/quote-server.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package quoteserver;
+option go_package = "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto/quoteServer";
```

Comment:
What's usage of "go_package"? I did not see any reference elsewhere in this PR..

Reply:
its required by protobuf used in golang scenario